code/modules/reagents/chemistry/reagents/drug_reagents.dm

```diff
@@ -107,7 +107,7 @@
 	overdose_threshold = 20
 	ph = 9
 	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
-	addiction_types = list(/datum/addiction/opiods = 18) //7.2 per 2 seconds
+	addiction_types = list(/datum/addiction/opioids = 18) //7.2 per 2 seconds
 
 
 /datum/reagent/drug/krokodil/on_mob_life(mob/living/carbon/M, delta_time, times_fired)
```

Comment:
not important, but while you're here, you could rename all those single letter var names

Reply:
probably a little out of scope for this pr